src/uhttp.c

```diff
@@ -307,9 +310,63 @@ static int process_header_line(const unsigned char* buffer, size_t len, size_t*
     return result;
 }
 
+static void log_data_line(HTTP_CLIENT_HANDLE_DATA* http_data, const char* text_line)
+{
+    char* authStart = strstr(text_line, AUTH_HEADER);
+    char* proxyAuthStart = strstr(text_line, PROXY_AUTH_HEADER);
+    char* authEol = NULL;
+    char* proxyAuthEol = NULL;
+
+    if (authStart != NULL)
+    {
+        authEol = strstr(authStart, HEADER_ENDING);
+    }
+    if (proxyAuthStart != NULL)
+    {
+        proxyAuthEol = strstr(proxyAuthStart, HEADER_ENDING);
+        authStart = strstr(proxyAuthEol, AUTH_HEADER);
```

Comment:
So here if proxyAuthStart is not NULL, you are overriding authStart?
In that case the previous code that sets or checks authStart is moot.

Reply:
Good point! Fixed.